mongo-dao/src/main/resourses/application.properties

```diff
@@ -1,2 +1 @@
-spring.data.mongodb.uri= mongodb+srv://Arundhathy:Ammu2002@kovan.us1gd.mongodb.net/?retryWrites=true&w=majority&appName=Kovan
-spring.data.mongodb.database = NewsDB
+spring.data.mongodb.uri= mongodb://localhost:27017/NewsDB
```

Comment:
why do we need this property file here ?

Reply:
property file removed